pkgs/top-level/all-packages.nix

```diff
@@ -588,6 +588,9 @@ with pkgs;
 
   crow-translate = libsForQt5.callPackage ../applications/misc/crow-translate { };
 
+  inherit (callPackage ../misc/crust { })
+    crustFirmwareOrangePi3LTS;
+
```

Comment:
Nit: if the package is gonna end up in `pkgs/misc` anyway, probably put it under `pkgs/by-name`. See:
- https://github.com/NixOS/rfcs/blob/master/rfcs/0140-simple-package-paths.md
- https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md

Reply:
I tried to put it under pkgs/by-name before, but the CI "Check pkgs/by-name" failed. Probably because this pr uses inherit derivation?

Reply:
Interesting... In any case, this is just a nitpick. I'm okay with this as is. I don't have commit access though, you have to get someone else to merge...